templates/about/release_cycles/k8s-eol.html

```diff
@@ -1,107 +1,107 @@
-            <td align='right'>Apr 2024</td>
-            <td align='right'>Dec 2024</td>
-          </tr>
-          <tr>
-            <td><strong>1.26.x</strong></td>
-            <td align='right'>Dec 2022</td>
-            <td align='right'>Dec 2023</td>
-            <td align='right'>Aug 2024</td>
-          </tr>
-          <tr>
-            <td><strong>1.25.x</strong></td>
-            <td align='right'>Sep 2022</td>
-            <td align='right'>Aug 2023</td>
-            <td align='right'>Apr 2024</td>
-          </tr>
-          <tr>
-            <td><strong>1.24.x</strong></td>
-            <td align='right'>May 2022</td>
-            <td align='right'>May 2023</td>
-            <td align='right'>Dec 2023</td>
-          </tr>
-          <tr>
-            <td><strong>1.23.x</strong></td>
-            <td align='right'>Dec 2021</td>
-            <td align='right'>Dec 2022</td>
-            <td align='right'>Aug 2023</td>
-          </tr>
-          <tr>
-            <td><strong>1.22.x</strong></td>
-            <td align='right'>Sep 2021</td>
-            <td align='right'>Sep 2022</td>
-            <td align='right'>May 2023</td>
-          </tr>
-          <tr>
-            <td><strong>1.21.x</strong></td>
-            <td align='right'>Apr 2021</td>
-            <td align='right'>May 2022</td>
-            <td align='right'>Dec 2022</td>
-          </tr>
-          <tr>
-            <td><strong>1.20.x</strong></td>
-            <td align='right'>Dec 2020</td>
-            <td align='right'>Dec 2021</td>
-            <td align='right'>Sep 2022</td>
-          </tr>
-          <tr>
-            <td><strong>1.19.x</strong></td>
-            <td align='right'>Sep 2020</td>
-            <td align='right'>Sep 2021</td>
-            <td align='right'>May 2022</td>
-          </tr>
-          <tr>
-            <td><strong>1.18.x</strong></td>
-            <td align='right'>Apr 2020</td>
-            <td align='right'>Apr 2021</td>
-            <td align='right'>Dec 2021</td>
-          </tr>
-          <tr>
-            <td><strong>1.17.x</strong></td>
-            <td align='right'>Dec 2019</td>
-            <td align='right'>Dec 2020</td>
-            <td align='right'>Sep 2021</td>
-          </tr>
-          <tr>
-            <td><strong>1.16.x</strong></td>
-            <td align='right'>Sep 2019</td>
-            <td align='right'>Sep 2020</td>
-            <td align='right'>Apr 2021</td>
-          </tr>
-          <tr>
-            <td><strong>1.15.x</strong></td>
-            <td>Jun 2019</td>
-            <td>Apr 2020</td>
-            <td>Dec 2020</td>
-          </tr>
-          <tr>
-            <td><strong>1.14.x</strong></td>
-            <td>Mar 2019</td>
-            <td>Dec 2019</td>
-            <td>Sep 2020</td>
-          </tr>
-        </tbody>
-      </table>
-    </div>
+  <div class="col-10" style="overflow: auto;">
+    <div class="u-sv4 u-hide--small" id="kubernetes-eol"></div>
```

Comment:
```suggestion
    <div class="p-section--shallow u-hide--small" id="kubernetes-eol"></div>
```
We are trying not to use the `u-sv*` classes, and instead use the appropriate padding container. I think in this case `p-section--shallow` should work fine. We can consult with design if needed.
This should apply to all the uses of `u-sv4` below also.